doc/radosgw/upgrade_to_jewel.rst

```diff
@@ -0,0 +1,37 @@
+=============
+RGW upgrade procedure to older version of jewel
```

Comment:
So is this the doc. for upgrading to jewel versions upto 10.2.4? Is this applicable for 10.2.4 as well?